src/features/collections/CollectionPanel.tsx

```diff
@@ -11,10 +11,11 @@ import { List } from 'features/common/List'
 const LIMIT = 100
 
 export interface CollectionPanelProps {
-  pk?: string | null
+  pk?: string | null,
+  testId?: string
 }
 
-export function CollectionPanel({ pk }: CollectionPanelProps) {
+export function CollectionPanel({ pk, testId }: CollectionPanelProps) {
```

Comment:
Can we just call this `id`

Reply:
I've just realised `id` is not used. Can we make sure unused variables are cleared up. Will add a ticket to lint and fail on this in the CI too.

Reply:
@GlitchGoddessQA can you remove the prop though, as it's not being used.